openlibrary/core/lending.py

```diff
@@ -494,7 +494,33 @@ def _get_ia_loan(identifier, userid):
     return ia_loan and Loan.from_ia_loan(ia_loan)
 
 
-def get_loans_of_user(user_key):
+def cache_user_loans(func):
+    cached_loans = func.cached_loans = {}
+
+    def cache_func(*args, **kwargs):
+        if (
+            kwargs.get('use_cache', True)
+            and args in cached_loans
+            and time.time() - cached_loans[args]['timestamp'] < 300
```

Comment:
Suggestion: We could use https://github.com/internetarchive/openlibrary/blob/master/openlibrary/utils/dateutil.py#L13 `utils.dateutils.MINUTE_SECS * 5` to demystify the magic `300` constant

```suggestion
            and time.time() - cached_loans[args]['timestamp'] < 300
```